spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -200,9 +201,9 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
     }
   }
 
-  def prepareEntity(tableIdentifier: TableIdentifier): AtlasEntityWithDependencies = {
+  def prepareEntity(tableIdentifier: TableIdentifier): AtlasReferenceable = {
     val tableName = tableIdentifier.table
-    val dbName = tableIdentifier.database.getOrElse("default")
+    val dbName = tableIdentifier.database.getOrElse(SparkUtils.getCurrentDatabase)
```

Comment:
This fixes a bug I found while testing: if we call "USE <db>" and execute queries without specifying db explicitly, objects will be bound to such db, but tableIdentifier.database is `None`. 

`SparkUtils.getCurrentDatabase` will retrieve the current database which current session is using, hence able to deal with the case.

I'm going to craft the individual patch - this change is needed for tests in this patch to work, so have to include the fix here. We can merge the patch first and rebase this later.

Reply:
#247